qiskit_experiments/measurement/discriminator/discriminator_experiment.py

```diff
@@ -0,0 +1,59 @@
+"""
+Standard Discriminator Experiment class.
+"""
+
+from qiskit_experiments.base_experiment import BaseExperiment
+
+from qiskit.circuit import QuantumCircuit
+from .discriminator_analysis import DiscriminatorAnalysis
+from typing import List, Optional, Union, Iterable
+from qiskit.qobj.utils import MeasLevel
+
+
+class DiscriminatorExperiment(BaseExperiment):
+    """Discriminator Experiment class"""
```

Comment:
How would this work with say 0, 1, and 2? I.e. if we want to use a discriminator for a qutrit experiment. Will there be a new experiment or should we make the current one more flexible?

Reply:
The ESP discriminator is being implemented here: https://github.com/coruscating/qiskit-experiments/pull/4
Because it requires pulse, we felt it's significantly different from the standard discriminator and so it's a completely different class.